app/static/js/search.js

```diff
@@ -34,7 +34,8 @@ function handleSearchResults($ajaxPromise) {
     })
     .always(function () {
       $searchButton.prop("disabled", false);
-      $resultsContainer.focus();
+      $resultsContainer[0].focus();
+      $resultsContainer[0].scrollIntoView();
```

Comment:
`focus()` only scrolls to the element as far as it needs to go to have a teeny tiny bit on screen, and that was kind of weird. `scrollintoview()` will put a bit more of it in front of people.